### ISSUE TITLE
Object Detection

### ISSUE THREAD

**Issue by ekarabulut** [keras-team/autokeras#925]
Hi,

I have a model implemented in Darknet Yolo. I'd like to port my model to AutoKeras in order to use its features like hyperparameter optimization. In the documentation, 'ObjectDetection' seems to be an upcoming feature whereas in the source code I can see an SSD implementation exists.

My question is that whether it is possible to use object detection with AutoKeras? If the answer is yes, can you point me to some documentation/tutorial/example so that I can use it and port my model to AutoKeras.

**Comment by haifeng-jin** [comment]
@ekarabulut The object detection module is only a pertained one. It doesn't support custom dataset for training. Currently, we don't have this module. This is in our plan. Will have it in the near future. Thank you.

**Comment by hanotrahul** [comment]
hi, @haifeng-jin  when will Autokeras  support  object detection for a custom dataset  for training

**Comment by haifeng-jin** [comment]
@yufei-12 is designing the APIs.
Hopefully, we will have a basic model in the next 2 months.
Yufei, do you have any updates on this issue?

Thanks.

**Comment by yufei-12** [comment]
Hi, I'm reading the code of autogluon. I'll get started soon.Thank you.




------------------&nbsp;Original&nbsp;------------------

**Comment by takeofuture** [comment]
So ObjectDetection models eventually supported, (I am not talking about pretrained_model,  I assume it is not the job for autokeras as AutoML which automatically search and find candidates for the models based on given data).    
I am assuming given training dat asimilar like COCO or so (class label + (x, y, h, w) ) * xxx, then it find the good model for object detection automatically.  Any update so far? I am looking forward to using it!

**Comment by haifeng-jin** [comment]
We are really trying hard to reach this feature. This will be the next one after we have time-series, and segmentation. Of course, we will focus on the performance on existing tasks and fixing important bugs even before those.

**Comment by haifeng-jin** [comment]
We are working on retina net for now. May add the rest after [KerasCV](https://github.com/keras-team/governance/pull/26) is available.

**Comment by GaneshAdam** [comment]
@haifeng-jin , 
Thanks for updates.
when would retinanet be available.
Thanks in adavance!

**Comment by omar16100** [comment]
Hi @haifeng-jin 

Any update on this?

**Comment by muhammetp** [comment]
how to use the current framework in object detection by adapting some bbox predictor? is it possible
